src/components/ProductList/ProductListNavigation/ProductListNavigation.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Dropdown from '../../shared/Dropdown';
+import Button from '../../shared/Button';
```

Comment:
As I suggested earlier, absolute imports for shared folder woud be nice

Reply:
So you would use it like:
`import { Button, Dropdown } from 'shared';`

Reply:
We agreed that I will do it later, so this PR could be merged.

Reply:
But you will research it, right?

Reply:
what about this instead?
`import { Button, Dropdown } from '../shared';`